homeassistant/components/modbus/const.py

```diff
@@ -63,6 +63,16 @@
 CONF_HVAC_ON_VALUE = "hvac_on_value"
 CONF_HVAC_OFF_VALUE = "hvac_off_value"
 CONF_HVAC_ONOFF_COIL = "hvac_onoff_coil"
+CONF_HVAC_ACTION_REGISTER = "hvac_action_register"
+CONF_HVAC_ACTION_COOLING = "state_cooling"
```

Comment:
```suggestion
CONF_HVAC_ACTION_COOLING = "action_cooling"
```
could we change these to have the `action` prefix instead? or is there a specific reason to use `state`?

Reply:
`state_` was used for the prefix as is it the same prefix also used by `CONF_HVAC_MODE_` and `CONF_FAN_MODE_`, which are similar in function.

Reply:
Probaly to add a suffix _action_ could enhance the readibility, in example state_action_drying or directly action_drying.. 
See for reference state_fan_hight or swing_mode_state_xxx..

Reply:
> `state_` was used for the prefix as is it the same prefix also used by `CONF_HVAC_MODE_` and `CONF_FAN_MODE_`, which are similar in function.

`CONF_FAN_MODE_*` has `fan` on the name too. I think it helps readability, and avoids mixing them up with the HVAC_MODEs.
I like the suggestion from @crug80 too. 
We could also do `action_state_*`, which is consistent with the `swing_mode_state*` ones.
So either `action_*`, `action_state_*`, or `state_action_*`, as we have both variations already.